app/Http/Controllers/Frontend/Transport/StatusController.php

```diff
@@ -37,6 +38,8 @@ public function updateStatus(Request $request): JsonResponse|RedirectResponse {
                                 'visibility' => StatusVisibility::from($validated['checkinVisibility']),
                             ]);
 
+            StatusUpdateEvent::dispatch($status->refresh());
```

Comment:
Does this StatusUpdateEvent take a changed destination in account? It _might_, but I'm not sure. When we write tests for this, this could be a special case to look out for.

Reply:
I tried it and it fired on a destination change, but yeah this should be a test case